Documentation/guides/building-apps/build-items.md

```diff
@@ -222,8 +222,36 @@ This is so MSBuild does not try to interpret them as actual file wildcards.
 NOTE: `*`, `?` and `.` will be replaced in the `BuildApk` task with the
 appropriate RegEx expressions.
 
+If the default file glob is too restrictive you can remove it by adding the 
+following to your csproj
+
+```
+<ItemGroup>
+	<AndroidPackagingOptionsExclude Remove="$([MSBuild]::Escape('*.kotlin_*')" />
+</ItemGroup>
+```
+
 Added in Xamarin.Android 13.1 and .NET 7.
 
+## AndroidPackagingOptionsInclude
+
+A set of file glob compatible items which will allow for items to be
+included from the final package. The default values are as follows
+
+```
+<ItemGroup>
+	<AndroidPackagingOptionsInclude Include="$([MSBuild]::Escape('*reflect.kotlin_builtins')" />
+</ItemGroup>
+```
+Items can use file blob characters for wildcards such as `*` and `?`.
+However these Items MUST use URL encoding or '$([MSBuild]::Escape(''))'.
+This is so MSBuild does not try to interpret them as actual file wildcards.
+
+NOTE: `*`, `?` and `.` will be replaced in the `BuildApk` task with the
+appropriate RegEx expressions.
+
+Added in .NET 9.
```

Comment:
If this fixes an issue, is there any chance we'd take this to .NET 8 servicing?

Reply:
It doesn't fix an issue, it is a new feature. So my guess is that won't qualify for backporting.

Reply:
Ok, I thought it might fix: https://github.com/xamarin/xamarin-android/issues/8456

If we have a way to include/exclude more or less files.

Reply:
It does fix #8456 in that it moves the the ItemGroup to  AutoImport.props, but the new ItemGroup is not required to fix that issue.